code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -10146,3 +10146,19 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	specheatcap = 0.45
 	density = 7.874
 	var/mute_duration = 300 //30 seconds
+
+/datum/reagent/fake_creep // Used to spread xenomorph creep. Why? Well, why not?
+	name = "Dan's Grape Drank"
+	id = FAKE_CREEP
+	description = "Discount Dan's award-winning grape drink. Limited production run! Now with added peanuts!"
```

Comment:
If it is discount dan's gunk, make it toxic and able to be made with dansauce, like how discount dan's malt whiskey is made with bleach and dansauce and has the same toxic effects of dansauce.